File: Angular/src/app/models/application-user.model.ts
export class ApplicationUserModel {
    id: string;
    firstName: string;
    lastName: string;
    profilePictureUrl: string;

    constructor(item: any) {
        this.id = item.id;
        this.firstName = item.firstName;
        this.lastName = item.lastName;
        this.profilePictureUrl = item.profilePictureUrl;
    }
}

File: ASP.NET API/WAVC_WebApi/Models/HelperModels/ApplicationUserModel.cs
using Microsoft.AspNetCore.Http;
using Microsoft.AspNetCore.Http.Internal;
using Microsoft.AspNetCore.StaticFiles;
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;

namespace WAVC_WebApi.Models.HelperModels
{
    public class ApplicationUserModel
    {
        public string Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string ProfilePictureUrl { get; set; }

        public ApplicationUserModel(ApplicationUser user)
        {
            Id = user.Id;
            FirstName = user.FirstName;
            LastName = user.LastName;
            ProfilePictureUrl = user.ProfilePictureUrl;
        }
        public ApplicationUserModel()
        {

        }
    }
}
